File: submissions/Esebame1/Week_1_solution.py
for a in range (1,100):
  if a%3==0 and a%5==0:
    print('Fizzbuzz\n')
  elif a%3==0:
    print('Fizz\n')
  elif a%5==0:
    print('Buzz\n')
  else:
    print (str(a)+'\n')

File: submissions/Danielaghotor/Week_1_Solution.java
public class FizzBuzzMain {
	public static void main(String[] args) {
		
		for ( int fb=1; fb<=100; fb++) {
			if ((fb%3==0) && (fb%5==0)) {
				System.out.println("FizzBuzz");
			}
			else if (fb%3==0) {
				System.out.println("Fizz");
			}
			else if (fb%5==0) {
				System.out.println("Buzz");
			}
		else
			System.out.println(fb);	
		}
	}
}
